config.toml

```
@@ -18,3 +18,8 @@ css = ["core", "list"]
 [[params.fonts]]
 name = "Roboto"
 sizes = [300,400,600,700]
+
+[menu]
+[[menu.sidebar]]
+name = "Code of Conduct"
+url = "https://github.com/cncf/foundation/blob/master/code-of-conduct.md"
```

```suggestion
url = "https://github.com/kubernetes/community/blob/master/code-of-conduct.md"
```
Small nit.
While that doc points back to the CNCF CoC, we reference "our" CoC in case they should ever diverge. We don't have to update the links everywhere etc.